.travis.yml

```diff
@@ -28,5 +28,9 @@ jobs:
     - stage: Format
       env:
       script: bash '.ci-scripts/format'
+    - stage: Test
+      name: Run Benches
+      env:
+      script: bash '.ci-scripts/bench'
 before_cache:
```

Comment:
:+1:

Reply:
But, the result only outputs into CI logs.
So, we have to read the result from CI logs.

Add into the CI, only for keep the codes are maintainable.